packages/eos-components/src/components/PackCard.vue

```diff
@@ -50,12 +44,20 @@
         default: '',
       },
     },
+    data() {
+      return {
+        isHovered: false,
+      };
+    },
     methods: {
       onClick() {
         this.$emit('click');
       },
-      onChoosePack() {
-        this.$emit('choosePack');
+      onMouseOver() {
+        this.isHovered = true;
+      },
+      onMouseLeave() {
+        this.isHovered = false;
```

Comment:
I'm unclear why we're doing this _this_ way, so it could use a quick explanation in the comments. It kind of looks like we could use [v-b-hover](https://bootstrap-vue.org/docs/directives/hover) to save some boilerplate, or even add `.card:hover` in our CSS?

Reply:
Oh right! I was copying code from another of our cards which do the same for adding these Bootstrap CSS classes. There is no other way of adding classes dynamically on the template side. But I went to the bootstrap code and found what these classes do:

```
.card {
  box-shadow: $box-shadow-sm !important;
  &:hover {
    box-shadow: $box-shadow !important;
  }
}
```